packages/vue/src/components/verify-user.vue

```diff
@@ -87,6 +84,7 @@ const onSkipClicked = (): void => {
           <base-wrapper
             class="amplify-flex amplify-field amplify-radiogroupfield amplify-authenticator__column"
           >
+            <!-- TODO(BREAKING): remove hard coded string 493c -->
```

Comment:
We should use utils such as `nanoid` to get unique ids instead of hard coded strings. Left as is because such change would be breaking.